docs/apis/kopflos/reference/configuration.md

```diff
@@ -0,0 +1,27 @@
+# Kopflos configuration
+
+Kopflos can be configured via a configuration file, which can be in various formats and locations. If not provided, it will try various default locations:
```

Comment:
why so many options? I guess it may help to make it _just work™_ but at the same time it may be confusing (and more work to implement :-)

Reply:
Becuase with [cosmiconfig](https://www.npmjs.com/package/cosmiconfig) it will require almost no work at all :)